main.go

```diff
@@ -119,6 +128,10 @@ func main() {
 		}
 
 		for name, value := range labelsFromSource {
+			// Skip if label doesn't match labelWhiteList
+			if !labelWhiteList.Match([]byte(name)) {
```

Comment:
I would like to see the feature logged even though the publish part is filtered by the whitelist. This could help operators to decide what to allow in the whitelist. Perhaps we could move the log before checking against the whitelist. Also, let's log filtered label names to stderr.